portal/templates/dashboard/_macros/dashboard-deals-table.njk

```diff
@@ -36,28 +37,31 @@
       {% set dealRow = [
         {
           text: deal.exporter or "-",
-          attributes: { "data-cy": "deal__exporter--" + deal._id  }
+          attributes: { "data-cy": "deal__exporter--" + deal._id + " deal__exporter--index--" + loop.index   }
         },
         {
           html: linkHtml,
-          attributes: { "data-cy": "deal__bankRef--bankInternalRefName" + deal._id }
+          attributes: { "data-cy": "deal__bankRef--bankInternalRefName" + deal._id +  " deal__bankInternalRefName--index--" + loop.index  }
         },
         {
           text: deal.product or "-",
-          attributes: { "data-cy": "deal__product--" + deal._id }
+          attributes: { "data-cy": "deal__product--" + deal._id + " deal__product--index--" + loop.index }
         },
         {
           text: deal.submissionType or "-",
-          attributes: { "data-cy": "deal__submissionType--" + deal._id }
+          attributes: { "data-cy": "deal__submissionType--" + deal._id + " deal__submissionType--index--" + loop.index }
         },
         {
           text: deal.status or "-",
-          attributes: { "data-cy": "deal__status--" + deal._id }
+          attributes: {
+            "data-cy": "deal__status--" + deal._id,
+            "id": "deal__status--index--" + loop.index
```

Comment:
Need to see if there is a better way of doing this, instead of adding an ID.

Otherwise, perhaps this will have to be temporary until the E2E tests for "multiple BSS/EWCS deals" (dashboard tests), are updated to work without IDs. Then, we can simply replace the data-cys with deal IDs, with indexes.